code/modules/clothing/factions/suns.dm

```diff
@@ -0,0 +1,572 @@
+	righthand_file = 'icons/mob/inhands/faction/suns/suns_righthand.dmi'
+
+/obj/item/clothing/head/safety_helmet/suns
+	desc = "A piece of headgear used in dangerous working conditions to protect the head."
+	icon_state = "suns_workerhelmet"
+	item_state = "suns_workerhelmet"
+	icon = 'icons/obj/clothing/faction/suns/head.dmi'
+	mob_overlay_icon = 'icons/mob/clothing/faction/suns/head.dmi'
+	lefthand_file = 'icons/mob/inhands/faction/suns/suns_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/faction/suns/suns_righthand.dmi'
+
+/obj/item/clothing/head/HoS/syndicate/suns //remind me to make this something to buy
+	name = "peacekeeper cap"
+	desc = "A black cap worn by the more eccentric peacekeepers."
+	icon_state = "suns_pkcap"
+	item_state = "suns_pkcap"
+	icon = 'icons/obj/clothing/faction/suns/head.dmi'
+	mob_overlay_icon = 'icons/mob/clothing/faction/suns/head.dmi'
+	lefthand_file = 'icons/mob/inhands/faction/suns/suns_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/faction/suns/suns_righthand.dmi'
+
+/obj/item/clothing/head/suns_surgery
+	name = "\improper SUNS surgery cap"
+	desc = "A surgery cap used by academic students and profesionals alike."
+	icon_state = "suns_doctorcap"
+	item_state = "suns_doctorcap"
+	icon = 'icons/obj/clothing/faction/suns/head.dmi'
+	mob_overlay_icon = 'icons/mob/clothing/faction/suns/head.dmi'
+	lefthand_file = 'icons/mob/inhands/faction/suns/suns_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/faction/suns/suns_righthand.dmi'
+
+/obj/item/clothing/head/welding/suns
+	name = "peacekeeper visor"
+	desc = "A head-mounted helmet designed to protect those on the field from bright lights, while also allowing a life support connection. The warnings on this helmet suggest it is not space proof."
+	icon_state = "sunsvisor"
+	item_state = "suns_pkhelmet"
+	tint = 0
+	clothing_flags = BLOCK_GAS_SMOKE_EFFECT | ALLOWINTERNALS //Why? Because I'm not giving PK's sec masks nor hud sunglasses.
+	icon = 'icons/obj/clothing/faction/suns/head.dmi'
+	mob_overlay_icon = 'icons/mob/clothing/faction/suns/head.dmi'
+	lefthand_file = 'icons/mob/inhands/faction/suns/suns_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/faction/suns/suns_righthand.dmi'
+
+/obj/item/clothing/head/welding/suns/hos
+	name = "gilded peacekeeper visor"
+	desc = "A head-mounted helmet designed to protect those on the field, this one has a gold lining to indicate rank. The warnings on this helmet suggest it is not space proof."
+	icon_state = "sunslpkvisor"
+	item_state = "suns_pkhelmet"
+
+/obj/item/clothing/head/sunscaptain
+	name = "\improper SUNS bicorne hat"
+	desc = "A unique bicorne hat given to SUNS Captains to display academic seniority."
+	icon_state = "suns_captainbicorne"
+	item_state = "suns_captainbicorne"
+	worn_y_offset = 2
+	dog_fashion = null
+	armor = list("melee" = 40, "bullet" = 30, "laser" = 30, "energy" = 40, "bomb" = 25, "bio" = 0, "rad" = 0, "fire" = 30, "acid" = 60)
```

Comment:
No armored hats. Needs to reflect its armor in the sprite if it will have armor

Reply:
This behavior is seen in every other command hat in the game, to my knowledge. While I agree, I think it would be odd to neuter SUNS uniquely in that regard. Nerfing every command hat (and some niche cases, like sonnensoldners) is beyond the scope of this PR. I can and will remove the armor on this and the XO hat if you want, but again, it would be an odd outlier that puts this faction at a unique disadvantage, rather than the norm.